ui/src/data_explorer/components/View.js

```diff
@@ -0,0 +1,77 @@
+import React, {PropTypes} from 'react'
+
+import Table from './Table'
+import AutoRefresh from 'shared/components/AutoRefresh'
+import LineGraph from 'shared/components/LineGraph'
+import SingleStat from 'shared/components/SingleStat'
+const RefreshingLineGraph = AutoRefresh(LineGraph)
+const RefreshingSingleStat = AutoRefresh(SingleStat)
+
+const View = ({
```

Comment:
I know this is scoped within `data_explorer`, but can we rename this to something more specific to its function? `View` is a bit too general and doesn't help to provide context.